handlers/aerospike/aerospike.go

```diff
@@ -0,0 +1,122 @@
+package aerospike
+
+import (
+	"errors"
+	"strconv"
+	"strings"
+
+	aero "github.com/aerospike/aerospike-client-go"
+	"github.com/netflix/rend/common"
+	"github.com/netflix/rend/handlers"
+)
+
+var (
+	errNotFound = "Key not found"
+)
+
+type Handler struct {
+	client    *aero.Client
+	namespace string
+	set       string
+}
+
+func NewHandler(clusterAddr string, bucketName string) (Handler, error) {
+	addr := strings.Split(clusterAddr, ":")
+	port, err := strconv.Atoi(addr[1])
+	if err != nil {
+		return Handler{}, err
+	}
+	client, err := aero.NewClient(addr[0], port)
+	if err != nil {
+		return Handler{}, err
+	}
+
+	return Handler{client: client, namespace: bucketName, set: "rend"}, nil
```

Comment:
I think that the bucket notion is more the set on aerospike

Reply:
Yes but it was a cheap way to be able adapt the namespace for testing (so I can change between ssd/nvme/persistent for my tests)

Reply:
ok